source/renderer/app/components/staking/epochs/helpers.js

```diff
@@ -0,0 +1,68 @@
+import orderBy from 'lodash/orderBy';
+import humanizeDuration from 'humanize-duration';
+
+export const SELECTED_EPOCH_OPTIONS = {
+  CURRENT_EPOCH: 'currentEpoch',
+  PREVIOUS_EPOCH: 'previousEpoch',
+};
+
+export const getTableHeadersForCurrentEpoch = (intl, messages) => [
+  {
+    name: 'pool',
+    title: intl.formatMessage(messages.tableHeaderPool),
+  },
+  {
+    name: 'slotsElected',
+    title: intl.formatMessage(messages.tableHeaderSlotsElected),
+  },
+];
+
+export const getTableHeadersForPreviousEpoch = (intl, messages) => [
+  {
+    name: 'pool',
+    title: intl.formatMessage(messages.tableHeaderPool),
+  },
+  {
+    name: 'slotsElected',
+    title: intl.formatMessage(messages.tableHeaderSlotsElected),
+  },
+  {
+    name: 'performance',
+    title: intl.formatMessage(messages.tableHeaderPerformance),
+  },
+  {
+    name: 'sharedRewards',
+    title: intl.formatMessage(messages.tableHeaderSharedRewards),
+  },
+];
+
+export const noDataExisting = (isLoading, data) =>
+  !isLoading && ((data && !data.length) || !data);
+
+export const hasDataExisting = (isLoading, data) =>
+  data && data.length > 0 && !isLoading;
+
+export const sortData = (data, order, sortBy) =>
+  orderBy(data, sortBy === 'pool' ? 'pool.title' : sortBy, order);
+
+export const humanizeDurationToShort = dateTime =>
```

Comment:
@yakovkaravelov wouldn't it be better to use `humanizeDuration` like you did here? https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/components/widgets/CountdownWidget.js#L76

Reply:
I am using new humanizeDuration format because wireframe shows short format of duration label.
What I did in countdownwidget is showing full length duration label.

Reply:
But this covers ONLY English 😢 
Isn't there a way to have this short format by modifying humanizer options?
If this is too complex perhaps we can change the format to whatever humanizer has as a shortest version. Please talk to @a-rukin to see how that would fit in.

Reply:
@yakovkaravelov any news on this one?